pyanaconda/ui/gui/spokes/custom.py

```diff
@@ -2007,16 +2007,20 @@ def run_container_editor(self, container=None, name=None):
         if not disks:
             self._error = "No disks selected. Not saving changes."
```

Comment:
Totally true if I didn't missed it would be fixed in this patch already. Thank you.